src/popup/index.js

```diff
@@ -1,35 +1,103 @@
 import React, { Component } from 'react';
-import _ from 'underscore';
+import _                    from 'underscore';
+import $                    from 'jquery';
+import classNames           from 'classnames';
+
+// Components
+import {ActionButton} from 'bilprospekt-ui';
+
+const Popup = React.createClass({
+    propTypes: {       
+        button: React.PropTypes.object.isRequired,
+        content: React.PropTypes.object.isRequired,
+        actionLabel: React.PropTypes.string.isRequired,
+        closeLabel: React.PropTypes.string.isRequired,
+        display: React.PropTypes.bool,
+    },
+
+    getInitialState() {
+        return {
+            display: false,
+            windowHeight: window.innerHeight,
```

Comment:
Why are you keeping window.innerHeight in state? It doesn't look like you ever modify it. Why not just rerender component when resize is triggered, and refer to window.innerHeight directly in didUpdate?